templates/admin/html-variation-price.php

```diff
@@ -73,3 +56,30 @@
 		</select>
 	</p>
 </div>
+<div class="variable_subscription_trial variable_subscription_pricing_2_3 show_if_variable-subscription variable_subscription_trial_sign_up" style="display: none">
+	<p class="form-row form-row-first form-field show_if_variable-subscription sign-up-fee-cell">
+		<label for="variable_subscription_sign_up_fee[<?php echo esc_attr( $loop ); ?>]">
+			<?php printf( esc_html__( 'Sign-up fee (%s)', 'woocommerce-subscriptions' ), esc_html( get_woocommerce_currency_symbol() ) ); ?>
+			<?php echo wcs_help_tip( _x( 'Optionally include an amount to be charged at the outset of the subscription. The sign-up fee will be charged immediately, even if the product has a free trial or the payment dates are synced.', 'woocommerce-subscriptions', 'woocommerce-subscriptions' ) ); ?>
```

Comment:
```suggestion
			<?php echo wcs_help_tip( _( 'Optionally include an amount to be charged at the outset of the subscription. The sign-up fee will be charged immediately, even if the product has a free trial or the payment dates are synced.', 'woocommerce-subscriptions' ) ); ?>
```

Similar to the comment above, this `_x()` did have the second param, however it wasn't offering any further context other than the text domain. In the other place where there's a similar string ([here](https://github.com/automattic/woocommerce-subscriptions-core/blob/8.1.0/includes/admin/class-wc-subscriptions-admin.php#L358)) we just use a `__()`.